src/Modal.js

```diff
@@ -119,30 +188,63 @@ const Modal = React.createClass({
   },
 
   componentDidMount() {
+    const doc = domUtils.ownerDocument(this);
+    const win = domUtils.ownerWindow(this);
+
     this._onDocumentKeyupListener =
-      EventListener.listen(domUtils.ownerDocument(this), 'keyup', this.handleDocumentKeyUp);
+      EventListener.listen(doc, 'keyup', this.handleDocumentKeyUp);
+
+    this._onWindowResizeListener =
+        EventListener.listen(win, 'resize', this.handleWindowResize);
```

Comment:
no real reason other then it struck me as easier to read  and was in line with the formatting pattern of the handler above it :)